client/src/components/Layout/rooms/Rooms.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from "react";
+import axios from "axios";
+import "./style.css";
+
+class Rooms extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      rooms: [],
+    };
+  }
+
+  componentDidMount() {
+    axios
+      .get("/api/rooms")
+      .then((res) => this.setState({ rooms: res.data }))
+      .catch((err) => console.log(err));
+  }
+
+  showInfo = (e) => {
+    const info = e.target.nextSibling;
+    
+    if (info === null || info.className === "li" ) return;
+    if (info.style.display === "block") {
+      info.style.display = "none";
+
+    } else {
+      info.style.display = "block";
+
+    }
+  };
+
+  render() {
+    const { rooms } = this.state;
+    return (
+      <ul className="rooms">
+        {rooms.map((rooms) => {
+          return (
+            <div>
+              <div onClick={this.showInfo} className="rooms__card">
```

Comment:
Okay, Its a good solution but not the best, avoid use CSS in react 
You can use prevState to toggle divs

Reply:
Done